checkov/common/bridgecrew/bc_source.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional
+
+# Helper methods for determining behavior based on different BC_SOURCE values. Python enums are limiting here, and
+# using a class is overkill
+
+
+def should_upload_results(source: Optional[str]) -> bool:
+    """Whether scan results should be uploaded to the platform. Default for unknown sources is False."""
+    if source == 'vscode':
+        return False
+    elif source == 'cli':
+        return True
+    else:
+        return False
```

Comment:
not sure, why there should be an extra `vscode` check?
```suggestion
    if source == 'cli':
        return True
    return False
```
or an `else` at the end, whichever style you prefer 🙂

Reply:
Because vscode is a valid source.

Reply:
But it will still return `False` like on the `else` case. So, there is no real benefit for checking the `source` is `vscode`. When you change the method, then all the tests will still pass.

Reply:
Fair enough, but I am using this as an ad hoc enum (since Python's enums seem limiting here), and it seems better to me to be explicit.

Reply:
I plan to take a dataclass approach that Nimrod showed me, to make it more explicit for each source

Reply:
Ok, sounds good.